resources/device_probe.py

```diff
@@ -595,6 +595,8 @@ class Computer:
     opencore_version: Optional[str] = None
     opencore_path: Optional[str] = None
     bluetooth_chipset: Optional[str] = None
+    keyboard_type: Optional[str] = None
```

Comment:
```suggestion
    internal_keyboard_type: Optional[str] = None
```
`keyboard_type` is vague